website/routes.py

```diff
@@ -907,6 +907,7 @@ def make_url_map(app):
 
         # Create project, used by projectCreator.js
         Rule('/project/new/', 'post', project_views.node.project_new_post, json_renderer),
+        Rule('/project/new/delete/<pid>/', 'delete', project_views.node.project_delete_new_post, json_renderer),
```

Comment:
This route seems unnecessary. Can you use the existing route for deleting nodes? https://github.com/CenterForOpenScience/osf.io/blob/e1a2d38ed8c9a8c2cd4b6d6126f1c176e9af8bcd/website/routes.py#L1101-1110